tools/ci_build/github/azure-pipelines/linux-rocm-ci-pipeline.yml

```diff
@@ -0,0 +1,238 @@
+variables:
+  - name: video
+    value: 44
+  - name: render
+    value: 109
+  - name: RocmVersion
+    value: 6.1
+  - name: RocmVersionPatchSuffix
+    value: ".3"
+
+jobs:
+- job: Linux_Build
+  variables:
+    skipComponentGovernanceDetection: true
+    CCACHE_DIR: $(Pipeline.Workspace)/ccache
+    TODAY: $[format('{0:dd}{0:MM}{0:yyyy}', pipeline.startTime)]
+  workspace:
+    clean: all
+  pool: onnxruntime-Ubuntu2204-AMD-CPU
+  timeoutInMinutes: 240
+
+  steps:
+  - task: mspremier.PostBuildCleanup.PostBuildCleanup-task.PostBuildCleanup@3
+    displayName: 'Clean Agent Directories'
+    condition: always()
+
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+
+  - template: templates/get-docker-image-steps.yml
+    parameters:
+      Dockerfile: tools/ci_build/github/linux/docker/rocm-ci-pipeline-env.Dockerfile
+      Context: tools/ci_build/github/linux/docker
+      DockerBuildArgs: "--build-arg ROCM_VERSION=$(RocmVersion)$(RocmVersionPatchSuffix)"
+      Repository: onnxruntimerocm-cibuild-rocm$(RocmVersion)
+
+  - task: Cache@2
+    inputs:
+      key: '"$(TODAY)" | "$(Build.SourceBranch)" | "$(Build.SourceVersion)"'
+      path: $(CCACHE_DIR)
+      cacheHitVar: CACHE_RESTORED
+      restoreKeys: |
+        "$(TODAY)" | "$(Build.SourceBranch)"
+        "$(TODAY)" |
+    displayName: Cache Task
+
+  - script: mkdir -p $(CCACHE_DIR)
+    condition: ne(variables.CACHE_RESTORED, 'true')
+    displayName: Create Cache Dir
+
+  - task: CmdLine@2
+    inputs:
+      script: |
+        docker run --rm \
+          --security-opt seccomp=unconfined \
+          --shm-size=1024m \
+          --user $UID:$(id -g $USER) \
+          --volume $(Build.SourcesDirectory):/onnxruntime_src \
+          --volume $(Build.BinariesDirectory):/build \
+          --volume $(CCACHE_DIR):/cache \
+          -e CCACHE_DIR=/cache \
+          --workdir /onnxruntime_src \
+          onnxruntimerocm-cibuild-rocm$(RocmVersion) \
+          /bin/bash -c "
+            set -ex; \
+            env; \
+            ccache -s; \
+            python tools/ci_build/build.py \
+              --config Release \
+              --cmake_extra_defines \
+                CMAKE_HIP_COMPILER=/opt/rocm/llvm/bin/clang++ \
+                onnxruntime_BUILD_KERNEL_EXPLORER=ON \
+                CMAKE_HIP_ARCHITECTURES=gfx90a \
+              --mpi_home /opt/ompi \
+              --use_rocm \
+              --rocm_version=$(RocmVersion) \
+              --rocm_home /opt/rocm \
+              --nccl_home /opt/rocm \
+              --enable_nccl \
+              --update \
+              --build_dir /build \
+              --build \
+              --build_shared_lib \
+              --parallel \
+              --build_wheel \
+              --enable_onnx_tests \
+              --skip_submodule_sync \
+              --use_cache \
+              --skip_tests --cmake_path /usr/bin/cmake --ctest_path /usr/bin/ctest; \
```

Comment:
Why it skip tests?

Reply:
We don't run tests in the build stage. They will be run and verified in test stage.

Reply:
Then you can do "--update --build"